kalitectl.py

```diff
@@ -571,6 +572,27 @@ def start(debug=False, watch=False, daemonize=True, args=[], skip_job_scheduler=
             'server.socket_port': port
         })
 
+    # idealy we'd have a setting element, but these are imported much later
+    # and it will require a much deeper change
+    idle_timeout = int(os.getenv("KALITE_IDLE_TIMEOUT", 0))
```

Comment:
what to do with 'Null' 'NO' 'False' and all other strings ? I will need to make the check, then int() then make another check. i don't see how it helps.

code will have another nested if... i don't see how it's more readable.